docs/references/nextjs/custom-signup-signin-pages.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "Create custom sign-in and sign-up pages for Next.js with Clerk"
+description: "Learn how to add custom sign-in and sign-up pages to your Next.js app with Clerk's prebuild components and custom flows."
```

Comment:
frontmatter doesn't need quotations.
title should be same as the leading h1 (there is a ticket to automate/address this)

```suggestion
title: Build your own sign-in and sign-up pages for your Next.js App with Clerk
description: Learn how to add custom sign-in and sign-up pages to your Next.js app with Clerk's prebuilt components and custom flows.
```

Reply:
With this content broken into its own section, this particular guide may fit better as a guide on moving from Account Portal to self-hosting components. Additions required for that would be:

- a /user or /profile page that mounts <UserProfile />
- disabling Account Portal
- if organizations are enabled for the application
- a /organization page that mounts <OrganizationProfile />
- a /create-organzation page that mounts <CreateOrganization />
- mounting <OrganizationSwitcher /> in the header inside the layout

This might be a bit much, but at the time once customers are moving to hosting their own /signin and /signup routes they are likely moving to this full solution anyway.